packages/dashboard/src/pages/users/[id].page.tsx

```diff
@@ -119,6 +122,46 @@ const User: NextPage<UserPageProps> = function User(props) {
     2
   );
 
+  const userDeleteRequest = useAppStore(
+    (store) => store.userDeleteRequest
+  );
+  const setUserDeleteRequest = useAppStore(
+    (store) => store.setUserDeleteRequest
+  );
+
+  const workspace = useAppStore((store) => store.workspace);
+  const apiBase = useAppStore((store) => store.apiBase);
+
+  const workspaceId =
+    workspace.type === CompletionStatus.Successful ? workspace.value.id : null;
+
+  const handleDelete = () => {
+    if (!workspaceId) {
+      return;
+    }
+
+    apiRequestHandlerFactory({
+      request: userDeleteRequest,
+      setRequest: setUserDeleteRequest,
+      responseSchema: EmptyResponse,
+      onSuccessNotice: `Deleted User`,
+      onFailureNoticeHandler: () =>
+        `API Error: Failed to delete User`,
+      setResponse: () => {},
```

Comment:
Here, can you redirect the user back to the `/users` page.?

Reply:
Sure @maxgurewitz , this is done.